app/src/ui/changes/sidebar.tsx

```diff
@@ -249,6 +249,14 @@ export class ChangesSidebar extends React.Component<IChangesSidebarProps, {}> {
     const fullPath = Path.join(this.props.repository.path, path)
     openFile(fullPath, this.props.dispatcher)
   }
+  /**
+   * Called to open a file in its the default external editor
```

Comment:
I noticed you left part of the typo here:
```suggestion
   * Called to open a file in the default external editor
```

Reply:
ha! You're patience knows no bounds.. I'll get it right eventually 😆 🤦🏽‍♂️

Reply:
LOL no worries 🫂 I make those mistakes (and much worse) all the time 😂